content/chainguard/chainguard-images/chainguard-registry/pull-through-guides/artifactory/artifactory-packages-pull-through/index.md

```diff
@@ -81,22 +121,19 @@ export ARTIFACTORY_HOSTNAME=my-artifactory-hostname
 If you aren't sure of these values, you can find them in the command from the **Set Up An Alpine Client** window where you found the token:
 
 ```shell
-sudo sh -c "echo 'https://linky:<TOKEN>@example-hostname.jfrog.io/artifactory/cg-wolfi/<BRANCH>/<REPOSITORY>'" >> /etc/apk/repositories
+sudo sh -c "echo 'https://linky:<TOKEN>@example-hostname.jfrog.io/artifactory/cg-private/<BRANCH>/<REPOSITORY>'" >> /etc/apk/repositories
```

Comment:
what is the branch and repository stuff about

Reply:
That's just what appears on Artifactory :shrug: